tests/testthat/test-geom-point.R

```diff
@@ -0,0 +1,32 @@
+context("translate_shape_string")
+
+test_that("strings translate to their corresponding integers", {
+  shape_strings <- c(
+    "square open",
+    "circle open",
+    "triangle open"
+  )
+
+  expect_equal(translate_shape_string(shape_strings[1]), 0)
+  expect_equal(translate_shape_string(shape_strings), 0:2)
```

Comment:
It's not immediately clear to me what the intent of these tests are — I think it's to check that `translate_shape_string()` is vectorised? If so, I think you should create a new test, and I think you should only need one expectation.